### PR TITLE
Fix Error Message Printout in Dependents Jobs

### PR DESCRIPTION
**Pull request by tfink419**
## Summary

- This work is behind a feature toggle (flipper): *NO*
- This fixes the issue we have been seeing in datadog where the error is wrapped in an additional "argument expected to be a string, got (ErrorObject).

## Related issue(s)

- *Link to ticket created in va.gov-team repo OR screenshot of Jira ticket if your team uses Jira*
- *Link to previous change of the code/bug (if applicable)*
- *Link to epic if not included in ticket*

## Testing done
- Verified that this error was occurring and is fixed when updating the sentry logging lines

## What areas of the site does it impact?
This only impacts logging

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
- [x]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x]  Feature/bug has a monitor built into Datadog (if applicable)
- [x]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [x]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by micahaspyr — CHANGES_REQUESTED** [review]
So log_message_to_sentry is being pushed out in favor of just using the Rails.logger.  You can just delete those two methods

File: app/sidekiq/bgs/submit_form674_job.rb [hunk 1]
```diff
@@ -36,7 +36,7 @@ def perform(user_uuid, icn, saved_claim_id, encrypted_vet_info, encrypted_user_s
 
       Rails.logger.warn('BGS::SubmitForm674Job received error, retrying...',
                         { user_uuid:, saved_claim_id:, icn:, error: e.message, nested_error: e.cause&.message })
-      log_message_to_sentry(e, :warning, {}, { team: 'vfs-ebenefits' })
+      log_message_to_sentry(e.message, :warning, {}, { team: 'vfs-ebenefits' })
```
[the remark this answers is not in the document]
Comment (micahaspyr) [reply app/sidekiq/bgs/submit_form674_job.rb:39]: Delete log_message_to_sentry

File: app/sidekiq/bgs/submit_form686c_job.rb [hunk 2]
```diff
@@ -36,7 +36,7 @@ def perform(user_uuid, icn, saved_claim_id, encrypted_vet_info)
 
       Rails.logger.warn('BGS::SubmitForm686cJob received error, retrying...',
                         { user_uuid:, saved_claim_id:, icn:, error: e.message, nested_error: e.cause&.message })
-      log_message_to_sentry(e, :warning, {}, { team: 'vfs-ebenefits' })
+      log_message_to_sentry(e.message, :warning, {}, { team: 'vfs-ebenefits' })
```
[the remark this answers is not in the document]
Comment (micahaspyr) [reply app/sidekiq/bgs/submit_form686c_job.rb:39]: Delete log_message_to_sentry

**Review by micahaspyr — APPROVED** [review]
LGTM

**Review by rjohnson2011 — APPROVED** [review]

**Pull request merged**